public/app/features/dashboard/api/ResponseTransformers.ts

```diff
@@ -383,14 +401,14 @@ function getVariables(vars: VariableModel[]): DashboardV2Spec['variables'] {
         const qv: QueryVariableKind = {
           kind: 'QueryVariable',
           spec: {
-            name: v.name,
-            label: v.label,
-            hide: transformVariableHideToEnum(v.hide),
-            skipUrlSync: Boolean(v.skipUrlSync),
+            ...commonProperties,
             multi: Boolean(v.multi),
             includeAll: Boolean(v.includeAll),
-            allValue: v.allValue,
-            current: v.current || { text: '', value: '' },
+            allValue: v.allValue ?? undefined,
```

Comment:
```suggestion
            ...{ allValue: v.allValue ?? undefined },
```
You can always do this, to remove undefined properties :)